Running:

```
pipenv shell
pipenv install
pipenv run python main.py
```

Current output:

> eventsourcing.exceptions.EventHashError
